app/namespace.go

```diff
@@ -1543,9 +1544,13 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 					}
 
 					namespace := ctx.String(NamespaceFlagName)
-					ns, err := c.getNamespace(namespace)
-					if err != nil {
-						return fmt.Errorf("unable to get namespace: %v", err)
+					region := ctx.String(RegionFlag.Name)
+					if len(region) == 0 {
```

Comment:
I was thinking of having the default be `ns.Spec.Region` and check if passiveregions length > 0 to fail early, but noticed that both Region and PassiveRegions are up to be deprecated (Region deprecated in the proto, looks like there are discussions to deprecate PassiveRegions here https://github.com/temporalio/saas-proto/pull/482/files#r1640308985),

Reply:
We could let this go through since this will fail on the validation for create export sink call anyways